src/main/java/frc/robot/subsystems/snapback/shooter/ShooterIOTalonFX.java

```diff
@@ -68,41 +80,49 @@ public ShooterIOTalonFX() {
 
     TalonFXConfiguration acceleratorConfig = rightFlywheelConfig;
     acceleratorConfig.CurrentLimits.SupplyCurrentLimit = 40.0;
-    acceleratorConfig.Feedback.SensorToMechanismRatio = ShooterConstants.ACCELERATOR_GEAR_RATIO;
+    acceleratorConfig.Feedback.SensorToMechanismRatio = ShooterConstants.ACCELERATOR_GEAR_REDUCTION;
     accelerator.getConfigurator().apply(acceleratorConfig);
 
     leftPosition = leftFlywheel.getPosition();
-    leftVelocityRadiansPerSecond = leftFlywheel.getVelocity();
+    leftVelocityRotationsPerSecond = leftFlywheel.getVelocity();
+    leftVelocitySetpointRadiansPerSecond = leftFlywheel.getClosedLoopReference();
+    leftVelocityErrorRadiansPerSecond = leftFlywheel.getClosedLoopError();
     leftAppliedVolts = leftFlywheel.getMotorVoltage();
     leftCurrentAmps = leftFlywheel.getSupplyCurrent();
     leftTemperatureCelsius = leftFlywheel.getDeviceTemp();
 
     rightPosition = rightFlywheel.getPosition();
-    rightVelocityRadiansPerSecond = rightFlywheel.getVelocity();
+    rightVelocityRotationsPerSecond = rightFlywheel.getVelocity();
+    rightVelocitySetpointRadiansPerSecond = rightFlywheel.getClosedLoopReference();
+    rightVelocityErrorRadiansPerSecond = rightFlywheel.getClosedLoopError();
     rightAppliedVolts = rightFlywheel.getMotorVoltage();
     rightCurrentAmps = rightFlywheel.getSupplyCurrent();
     rightTemperatureCelsius = rightFlywheel.getDeviceTemp();
 
     acceleratorPosition = accelerator.getPosition();
-    acceleratorVelocityRadiansPerSecond = accelerator.getVelocity();
+    acceleratorVelocityRotationsPerSecond = accelerator.getVelocity();
     acceleratorAppliedVolts = accelerator.getMotorVoltage();
     acceleratorCurrentAmps = accelerator.getSupplyCurrent();
     acceleratorTemperatureCelsius = accelerator.getDeviceTemp();
 
     BaseStatusSignal.setUpdateFrequencyForAll(
         50,
         leftPosition,
-        leftVelocityRadiansPerSecond,
+        leftVelocityRotationsPerSecond,
         leftAppliedVolts,
         leftCurrentAmps,
         leftTemperatureCelsius,
+        leftVelocitySetpointRadiansPerSecond,
+        leftVelocityErrorRadiansPerSecond,
         rightPosition,
-        rightVelocityRadiansPerSecond,
+        rightVelocityRotationsPerSecond,
         rightAppliedVolts,
         rightCurrentAmps,
         rightTemperatureCelsius,
+        rightVelocitySetpointRadiansPerSecond,
+        rightVelocityErrorRadiansPerSecond,
         acceleratorPosition,
-        acceleratorVelocityRadiansPerSecond,
+        acceleratorVelocityRotationsPerSecond,
         acceleratorAppliedVolts,
         acceleratorCurrentAmps,
         acceleratorTemperatureCelsius);
```

Comment:
Make sure to set the status signal parameters in the same order that the inputs are created in ShooterIO.java